src/utils/i18n.ts

```diff
@@ -1,7 +1,7 @@
 import en from '../_locales/en/messages.json'
 import zh from '../_locales/zh/messages.json'
-import React from 'react'
 import { GetContext } from '@holoflows/kit/es'
+import { safeReact, safeGetActiveUI } from './safeRequire.js'
```

Comment:
should it be './safeRequire(.ts)'?

Reply:
Oh that's strange. Import .ts file with a .js extname is valid in typescript. It must be there're some wrong configuration in our webpack config.